doc/FAQ.md

```diff
@@ -0,0 +1,142 @@
+# FAQ
+
+## Table of contents
+- [General questions](#general-questions)
+- [Loss of {funds / data}](#loss)
+
+
+## General questions
+
+### I don't know where to start, help me !
+
+Luckily, there is a C-lightning plugin specifically for this purpose, it's called
+[`helpme`](https://github.com/lightningd/plugins/tree/master/helpme).
+
+Assuming you already followed the installation step, have `lightningd` up and running, and
+`lightning-cli` in your `$PATH` you can start the plugin like so:
+
+```
+# Clone the plugins repository
+git clone https://github.com/lightningd/plugins
+# Make sure the helpme plugin is executable (git should have already handled this)
+chmod +x plugins/helpme/helpme.py
+# Install its dependencies (there is only one actually)
+pip3 install --user -r plugins/helpme/requirements.txt
+# Then just start it :)
+lightning-cli plugin start $PWD/plugins/helpme/helpme.py
+```
+
+The plugin registers a new command `helpme` which will guide you through the main
+components of C-lightning:
+
+```
+lightning-cli helpme
+```
+
+### How to get the balance of each channel ?
+
+You can use the `listfunds` command and make a ratio of `our_amount_msat` over `amount_msat`,
+though this doesn't take the reserve into account.
+
+You would be better just using the [`summary` plugin](https://github.com/lightningd/plugins/tree/master/summary)
+which nicely wraps these informations, along with others useful ones.
+
+### My channel is in state `STATE`, what does that mean ?
+
+See the [listpeers command manpage](https://lightning.readthedocs.io/lightning-listpeers.7.html#return-value).
+
+### My payment is failing / all my payments are failing, why ?
+
+There are a lot of possible reasons for a payment failure, but the most common one is a
+failure along the route from you to the payee.
+The best (and also common) answer to this problem is probably "open more channels",
+which will increase the number of routes to the recipient, hence lowering the overall
+probability of a failure.
+
+Also note to use the `pay` command which is trying all the routes for you, instead of
+the low-level `sendpay` command.
+
+### How can I receive payments ?
+
+In order to receive payments you need some incoming liquidity, which you can gain by paying
+through the channels you opened or by having other peers to open some channels to you.
+
```

Comment:
For example, you can use a service that exchanges Lightning Bitcoins for onchain Bitcoins (so that you do not lose your Bitcoins) in order to get incoming liquidity.  There also exist services that will, for a fee or for free, create channels to you.

Reply:
Yep i know about them but i was not sure about promoting them in our faq

Reply:
No need to name names (they may not be around forever anyway, so ---), at least it gives a hint to the user to search for such services by themselves.

Reply:
Yes, you are right.